.gdb_history

```diff
@@ -0,0 +1,6 @@
+start
```

Comment:
Please delete this file. It's ok to add `.gdb_history` to the .gitignore.

Reply:
Yikes! Goof on my part